ports/nrf/mpconfigport.mk

```diff
@@ -45,6 +45,9 @@ MCU_SERIES = m4
 MCU_VARIANT = nrf52
 MCU_SUB_VARIANT = nrf52840
 
+# Fits on nrf52840 but space is tight on nrf52833.
```

Comment:
Shouldn't you disable for the '833 here or does it fit for now?

Reply:
im pretty sure @xobs needs it

Reply:
I thought so too but the "but" in the comment seems to imply it doesn't fit on '833.

Reply:
ok this is by sean's request so we'll see what he says :)

Reply:
The `CIRCUITPY_AESIO ?= 1` is inside an `ifeq` that turns it on only for nrf52840. This is not obvious from the GitHub diff.

Reply:
@xobs turned it on explicitly in the `simmel/mpconfigboard.mk`, but it's not on in the other '833 boards.